### PR TITLE
Tuxcell pspincalc

### PR DESCRIPTION
**Pull request by tuxcell**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pspincalc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8b5f36e72f90ea7b494d63248117f62b0895a019a955ece13f52a22b65b09d2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/pspincalc/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/pspincalc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8b5f36e72f90ea7b494d63248117f62b0895a019a955ece13f52a22b65b09d2
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/pspincalc/meta.yaml]: Can it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```

File: recipes/pspincalc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - numpy
+    - PSpincalc
+
+about:
+  home: https://github.com/tuxcell/PSpincalc
+  license: GNU General Public v3 or later (GPLv3+)
+  license_family: GPL3
```
Comment (marcelotrevisani) [inline recipes/pspincalc/meta.yaml]:
```suggestion
  license_family: GPL
```

File: recipes/pspincalc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - numpy
+
+test:
+  imports:
+    - numpy
+    - PSpincalc
+
+about:
+  home: https://github.com/tuxcell/PSpincalc
+  license: GNU General Public v3 or later (GPLv3+)
```
Comment (marcelotrevisani) [inline recipes/pspincalc/meta.yaml]:
```suggestion
  license: GNU-3.0+
```

File: recipes/pspincalc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: GPL3
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/pspincalc/meta.yaml]: Could you please add the ``license_file``? You can package the license manually
Please see:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Comment (tuxcell) [reply recipes/pspincalc/meta.yaml]: Hi, yes, of course, it is done now. Thank you so much for your help! :)

Comment (tuxcell) [reply recipes/pspincalc/meta.yaml]: OK, now it seems to be complete.

File: recipes/pspincalc/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy
```
Comment (chrisburr) [inline recipes/pspincalc/meta.yaml]: Do you need `numpy` here? Unless it's imported by the `setup.py` script you probably don't
```suggestion
```

File: recipes/pspincalc/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - numpy
```
Comment (chrisburr) [inline recipes/pspincalc/meta.yaml]: Only test imports for things this package provides
```suggestion
```

Comment (tuxcell) [reply recipes/pspincalc/meta.yaml]: OK, I removed it now.

**Pull request merged**